eos-phone-home

```diff
@@ -70,9 +70,10 @@ import requests
 log = logging.getLogger(__name__)
 
 
-class PhoneHome(object):
-    STATE_DIRECTORY = "/var/lib/eos-phone-home"
+ProductInfo = collections.namedtuple('ProductInfo', 'vendor product serial')
```

Comment:
Nit: Use an array to define the tuple's props - single string implies a single field (even though it's syntactically correct).

Reply:
Ok.

Reply:
I will also correct the existing use of the space-separated string style in this file.